pkg/util/docker/docker_util.go

```diff
@@ -155,17 +158,12 @@ func (d *DockerUtil) dockerContainers(cfg *ContainerListConfig) ([]*Container, e
 			// FIXME: We might need to invalidate this cache if a containers networks are changed live.
 			d.Lock()
 			if _, ok := d.networkMappings[c.ID]; !ok {
-				// FIXME: Use du.Inspect instead
-				i, err := d.cli.ContainerInspect(ctx, c.ID)
-				if err != nil && client.IsErrContainerNotFound(err) {
+				i, err := d.Inspect(c.ID, false)
+				if err != nil {
 					d.Unlock()
 					log.Debugf("Error inspecting container %s: %s", c.ID, err)
 					continue
 				}
-				if i.ContainerJSONBase == nil {
```

Comment:
`d.Inspect` already checks for this. Added in https://github.com/DataDog/datadog-agent/pull/1684 as a quick fix for 6.2.0